pkg/common-controller/snapshot_controller.go

```diff
@@ -427,9 +458,25 @@ func (ctrl *csiSnapshotCommonController) syncUnreadySnapshot(snapshot *crdv1.Vol
 			ctrl.updateSnapshotErrorStatusWithEvent(snapshot, v1.EventTypeWarning, "SnapshotStatusUpdateFailed", fmt.Sprintf("Snapshot status update failed, %v", err))
 			return err
 		}
+
+		snapshotStatusType = metrics.SnapshotStatusTypeSuccess
 		return nil
 	}
-	// snapshot.Spec.Source.VolumeSnapshotContentName == nil - dynamically creating snapshot
```

Comment:
Can you not to remove this comment?

Reply:
Oops, thought I added this when I added the metric. Just re-added.